### ISSUE TITLE
Controller Mechanical Ventilation Availability Schedule

### ISSUE THREAD

**Issue by kbenne** [NREL/OpenStudio#2223]
User reports...

> I was trying to set an availability schedule for the ControllerMechanicalVentilation object in my translator program to disable the mechanical ventilation controller, but found in the generated IDF file that the availability schedule for the ControllerMechanicalVentilation object always uses the “Minimum Outdoor Air Schedule” of its associated “Controller:OutdoorAir” object. Looks like that is done by the forward translator of the ControllerMechanicalVentilation (https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/energyplus/ForwardTranslator/ForwardTranslateControllerMechanicalVentilation.cpp ,  lines 66-70 ).
>  
> Is it possible to change the forward translator code so that the forward translator does not automatically override the availability schedule I set for the controllerMechanicalVentilation?

Yes I think it is possible. Tricky though. I don't like the magic happening on lines 66-70, but it is there as a result of [this E+ issue](https://github.com/NREL/EnergyPlus/issues/4633). 

I propose that we make a change to OS and go ahead and translate the ControllerMechanicalVentilation::AvailabilitySchedule as given *provided* there is not a ControllerOutdoorAir::MinimOutdoorAirSchedule given *AND* the ControllerMechanicalVentilation availability is not still set to the default always on schedule.

Basically if a user provides any ControllerMechanicalVentilation::AvailabilitySchedule other than the default always on schedule, use what is given no matter what. This probably adds slightly more trickiness to already opaque logic, but I think it captures what people want to do without them having to know an obnoxious amount of detail.
